### ISSUE TITLE
[Bug]: Iron Command Block Sword Crash (possibly other items as well)

### ISSUE THREAD

**Issue by TeeJayMac** [nonamecrackers2/crackers-wither-storm-mod#2077]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Minecraft Version

1.20.1

### Mod Version

4.1

### Mod Loader

Forge

### Mod Loader Version

47.2.17 

### Describe the bug

I was messing around in a pretty large modpack i made (231 mods) and when i put the iron command block sword in my inventory, my game crashed.

### Steps to Reproduce

1. Go to the creative inventory
2. Put the iron command block sword into your hotbar
3. The game crashes

### Crash Reports/Logs

https://mclo.gs/hkf4bhP

### Is OptiFine installed?

- [ ] OptiFine is installed

### Mod List

![image](https://github.com/user-attachments/assets/9b8386bd-2a7b-489c-999c-56481d20564a)
![image](https://github.com/user-attachments/assets/ff28e85e-5f4b-4fb0-88ae-d4c3441347e0)

**Comment by xg787** [comment]
good ending is causing the issue remove it until they fix it on there end

**Comment by nonamecrackers2** [comment]
Yeah that appears to be the issue. Try removing that mod, and if that doesn't work, mind sending the crash report file?
